tests/pretty_format_kotlin_test.py

```diff
@@ -28,23 +29,21 @@ def undecorate_method():
 @pytest.mark.parametrize(
     ("filename", "expected_retval"),
     (
-        ("valid.kt", 0),
         ("invalid.kt", 1),
+        ("pretty-formatted.kt", 0),
+        ("not-pretty-formatted.kt", 1),
+        pytest.param(
+            "not-pretty-formatted_fixed.kt",
+            0,
+            marks=pytest.mark.xfail(
+                reason="Unclear why is not pretty-formatted but " "it is the result of pretty-fixing the file",
```

Comment:
```suggestion
                reason="Unclear why is not pretty-formatted but " " it is the result of pretty-fixing the file",
```

nit: the failure in the Kotlin file is

```
fun main(args : Array<String>) {
             ^
```
As per the Kotlin guidelines, colon should be just after parameter name (and not with a space).

Reply:
Found the reason of the ambiguity (`not-pretty-formatted_fixed.kt` is not pretty in `test_pretty_format_kotlin` but it is in `test_pretty_format_kotlin_autofix`).
This was caused by `.editorconfig` file present in the repo, it was specifying `insert_final_newline=false` which is read and parsed by ktlint (while running `test_pretty_format_kotlin`) but is not read while running `test_pretty_format_kotlin_autofix` as the `.editorconfig` file is not copied in the tmp directory.

To simplify it I've just removed the configuration that was defined most probably for legacy reasons ;)